roles/iofiller/tasks/main.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# (c) Copyright IBM Corporation
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Copy neerali_iofiller Container files
+  ansible.builtin.copy:
+    src: files
+    dest: "{{ neerali_iofiller_artifactdir }}"
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
+
+- name: Copy repo files
+  ansible.builtin.copy:
+    src: "/etc/yum.repos.d"
+    dest: "{{ neerali_iofiller_artifactdir }}/files"
+    remote_src: yes
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
```

Comment:
Do we need this step?
Can we explore `--volume` during `podman build`?

Reply:
Yes, I have tested volume and it works!!